api/shared-modules/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@iota-is/shared-modules",
+  "version": "0.1.0",
+  "description": "Shared modules of the Integration Service.",
+  "main": "index.js",
+  "scripts": {
+    "build-tsc": "tsc --project ./",
+    "build-lint": "eslint src --ext .js,.jsx,.ts,.tsx",
+    "build": "npm-run-all build-tsc build-lint",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "integration-services"
+  ],
+  "author": "Dominic Zettl (IOTA Foundation)",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@iota/identity-wasm": "^0.4.2",
+    "@sinclair/typebox": "^0.23.2",
+    "express": "^4.17.1"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.11",
+    "@types/jest": "^27.0.2",
+    "@typescript-eslint/eslint-plugin": "^4.14.1",
```

Comment:
@dominic22 is there any way to solve this? Have you the same warning?

Reply:
yes I have the same warning.. also for the api. I've pinned it to `"typescript": "=4.1.6"` otherwise it would always take the latest available cause of ^